Samples for Extending Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/src/main/java/org/pentaho/platform/spring/security/saml/PentahoSamlUserDetailsService.java

```diff
@@ -206,11 +215,11 @@ public void setSelectedAuthorizationProvider( String selectedAuthorizationProvid
     this.selectedAuthorizationProvider = selectedAuthorizationProvider;
   }
 
-  public UserDetailsService getSamlUserDetailsService() {
+  public PentahoSamlNativeUserDetailsService getSamlUserDetailsService() {
     return samlUserDetailsService;
   }
 
-  public void setSamlUserDetailsService( UserDetailsService samlUserDetailsService ) {
+  public void setSamlUserDetailsService( PentahoSamlNativeUserDetailsService samlUserDetailsService ) {
```

Comment:
@pedrofvteixeira Same comment as before. This delegating UserDetailsService has to know specifically about the SAML implementation of the UserDetailsService in case the user specifies authorization.provider=saml in pentaho.saml.cfg